guides/common/modules/proc_republishing-repository-metadata.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Republishing_Repository_Metadata_{context}"]
+= Republishing Repository Metadata
+
+Republishing repository metadata is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.
```

Comment:
Please reword to "You can use" or similar. This is passive voice.

Reply:
Changed to `You can republish repository metadata when a repository distribution does not have the content that should be distributed based on the contents of the repository.`